docker/Dockerfile-quickstart

```diff
@@ -0,0 +1,15 @@
+# This Dockerfile governs the uPortal Quickstart Docker image.  The portal must be fully assembled
+# (based on tomcatInstall + tomcatDeploy) before building the Docker image.
+
+# This image requires Gradle;  it is not as slim as the web server-only image
+FROM apereo/uportal-cli
+
+# The behavior of this image is based on a shell script
+COPY etc/docker/quickstart.sh quickstart.sh
+RUN chmod +x quickstart.sh
```

Comment:
can the file be `chmod`ed in the host/in source control and be copied as an executable into docker?

Reply:
I think so -- and it appears to work.  I cam across some documentation that seemed to suggest it should be done this way.